src/plot_erpimage.jl

```diff
@@ -13,7 +10,7 @@ Plot an ERP image.
 ## Keyword Arguments
 - `erpblur` (Number, `10`): number indicating how much blur is applied to the image; using Gaussian blur of the ImageFiltering module.
 Non-Positive values deactivate the blur.
-- `sortix` (default: `nothing`): .
+- `sortix` (default: `nothing`): ???.
```

Comment:
sortix allows to specify the ordering manually.

Reply:
What do you mean by manually? How it differs form sortvalues?

Reply:
sortix = [3,4,1,5,2]
sortval = [110 120 50 200 30]

Reply:
I would rather rename it then, because current name is kinda meaningless

Reply:
sure, sortindex and sortvalues is better anyway - I think sortindex is a good name though